docs/book/src/reference/envtest.md

```diff
@@ -167,5 +183,32 @@ expectedOwnerReference := v1.OwnerReference{
 Expect(deployment.ObjectMeta.OwnerReferences).To(ContainElement(expectedOwnerReference))
 ```
 
-[envtest]:https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/envtest
-[setup-envtest]:https://pkg.go.dev/sigs.k8s.io/controller-runtime/tools/setup-envtest
+## Cert-Manager and Prometheus options
+
+Projects scaffolded with Kubebuilder can enable the [`metrics`][metrics] and the [`cert-manager`][cert-manager] options. Note that when we are using the ENV TEST we are looking to test the controllers and their reconciliation. It is considered an integrated test because the ENV TEST API will do the test against a cluster and because of this the binaries are downloaded and used to configure its pre-requirements, however, its purpose is mainly to `unit` test the controllers.
+
+Therefore, to test a reconciliation in common cases you do not need to care about these options. However, if you would like to do tests with the Prometheus and the Cert-manager installed you can add the required steps to install them before running the tests. 
+Following an example.
+
+```go
+BeforeEach(func() {
+	By("installing the cert-manager bundle")
+	Expect(kbc.InstallCertManager(false)).To(Succeed())
+
+	By("installing the Prometheus operator")
+	Expect(kbc.InstallPrometheusOperManager()).To(Succeed())
+})
+```
```

Comment:
BEcause the links use Kubebuilder LIB for e2e tests to help out to mock things on the e2e tests done in the repo which are not good for these ones.